contributing_to_docs/doc_guidelines.adoc

```diff
@@ -42,24 +38,41 @@ Please set your editor to strip trailing whitespace.
 
 [NOTE]
 ====
-The topic title, which is the first line of the document, is the only level 1 ( = ) title. Section headers within the topic must be level 2 ( == ) or lower.
+The topic title, which is the first line of the document, is the only level 1 ( = ) title.
+Section headers (modules) within the topic/assembly must be level 2 ( == ) or lower, by using leveloffsets in include statements.
```

Comment:
Section headers within the assembly must be level 2 ( == ) or lower. When you include modules, you must add leveloffsets in the include statements. You can manually add more level 2 or lower section headers in the assembly.